RosBE-Unix/Base-i386/scripts/RosBE.sh

```diff
@@ -30,11 +30,6 @@ export BISON_PKGDATADIR="$_ROSBE_ROSSCRIPTDIR/share/bison"
 # The file "RosBE-Version" has been created by the RosBE-Builder.sh script
 export _ROSBE_VERSION=`cat "$_ROSBE_ROSSCRIPTDIR/RosBE-Version"`
 
-# Set the text color
```

Comment:
Why did you remove that? It's an optional parameter and may be used by people, who want the exact same experience that RosBE-Windows provides ;)